src/App.js

```diff
@@ -91,6 +77,25 @@ class App extends Component {
       </React.Fragment>
     );
   }
+
+  componentDidMount() {
+    getMovies()
+      .then((response) => this.setState({ movies: response.movies }))
+      .then(this.setState({ loader: false }))
```

Comment:
If the loading message is dependent solely on if there are movies rendered or not, we can remove this part of state and just check on this.state.moves! However if more than one condition can set this to true, then it can stay.

Reply:
duly noted!